index.html

```diff
@@ -1,38 +1,72 @@
 <!DOCTYPE html>
 <html>
+
 <head>
     <title>SongSeeker</title>
     <script type="module">
         import QrScanner from "https://unpkg.com/qr-scanner/qr-scanner.min.js";
     </script>
     <link rel="stylesheet" href="style.css">
+    <meta name=viewport content="width=device-width, initial-scale=1">
 </head>
+
 <body>
+    <h2>SongSeeker</h2>
+
     <div id="qr-reader">
+        <button id="cancelScanButton" class="button">Cancel Scan</button>
         <video id="qr-video"></video>
     </div>
-    <div id="qr-reader-results"></div>
-    <div>Playback Duration (seconds): <input type="number" id="playback-duration" value="30"></div>
-    <button id="set-duration">Set Duration</button>
-    
 
-    <div id="player"></div>
+    <button id="startScanButton" class="button_startscan">Start Scan</button>
+    <br />
+    <button id="startstop-video" class="button_startstop">Play</button>
+
+    <div id="videoid">Video ID: <span id="video-id"></span></div>
     <div id="videotitle">Video Title: <span id="video-title"></span></div>
-    <div>Video Duration: <span id="video-duration"></span></div>
-    <button id="play-video" class="button">Play all</button>
-    <button id="play-video-random-start" class="button">Play Random time</button>
-    <button id="stop-video" class="button">Stop</button>
-    <br>
-    <br>
-    <button id="startScanButton" class="button">Start Scan</button>
-    <button id="cancelScanButton" class="button">Cancel Scan</button>
-    <button id="hideInfo" class="button">Hide Song Info</button>
+    <div id="videoduration">Video Duration: <span id="video-duration"></span></div>
+
+    <hr />
 
     <div>
         <a href="https://github.com/andygruber/songseeker/" target="_blank">View the SongSeeker project on GitHub</a>
     </div>
 
+    <input type="checkbox" id="cb_settings" name="cb_settings" />
+    <label for="cb_settings">Show / Hide settings</label>
+    <div class="settings_div" id="settings_div">
+        <div class="text-block">
+            <div class="heading">Autoplay</div>
+            <div>
+                <input type="checkbox" id="autoplay" name="autoplay" checked />
+                <label for="autoplay">Autoplay</label>
+            </div>
+        </div>
+
+        <div class="text-block">
+            <div class="heading">Random Playback</div>
+            <div>
+                <input type="checkbox" id="randomplayback" name="randomplayback" />
+                <label for="randomplayback">Random Playback</label>
+                <br />
+                <label for="playback-duration">Playback Duration (seconds): </label>
+                <input type="number" id="playback-duration" value="30">
+            </div>
+        </div>
+
+        <div class="text-block">
+            <div class="heading">Debug</div>
+            <div>
+                <input type="checkbox" id="songinfo" name="songinfo" />
+                <label for="songinfo">Show song info</label>
```

Comment:
I liked the song info as indication, that a song is found.
It would be great, if this is disabled, that a generic infotext is shown, that a song is found.
With slow internet connection you might wait a bit for auto-playback to kick in, so at least you know, it tries to play something.

Reply:
As the Play button switches from Stop to play when the player is running there is some slight feedback, this might be improved with the button changing colour maybe from blue to green?.

Reply:
Some visible feedback would be much appreciated

Reply:
have some other idea for visible feedback, lets merge this now